pkgs/io_file/test/rename_test.dart

```diff
@@ -27,7 +27,7 @@ void main() {
       final path2 = '$tmp/file2';
 
       File(path1).writeAsStringSync('Hello World');
-      PosixFileSystem().rename(path1, path2);
+      fileSystem.rename(path1, path2);
```

Comment:
above, at the start of main(), perhaps assign the filesystem to a local? `final fs = fileSystem;`.

Reply:
Any reason why? Are you concerned with performance?

Reply:
No, mostly for documentation purposes in terms of what filesystem is being tested.